src/commands/android/index.ts

```diff
@@ -415,6 +415,10 @@ export class AndroidSetup {
       }
     }
 
+    if (options.standalone && options.browsers !== true && !configs.browsers) {
+      configs.browsers = 'none';
+    }
+
```

Comment:
Just to make it more readable.

```suggestion
    if (options.standalone && !configs.browsers) {
      configs.browsers = 'none';

      // if just the `--browsers` flag is passed with no argument,
      // ask the browser question even in the case of standalone.
      if (options.browsers === true) {
        delete configs.browsers;
      }
    }

```